### ISSUE TITLE
Could you please disclose the hardware configuration and detailed training parameter information used by you to train the COCO dataset and DOTAv1.0 dataset using v8 and v11?

### ISSUE THREAD

**Issue by JL-52Hertz** [ultralytics/ultralytics#22081]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I've reviewed some previous issues, but they're quite confusing and I'm unclear on the specific details regarding training v8 and v11 models. Could you please share the hardware configuration and detailed training parameters you used to train COCO and DOTA v1.0 using v8 and v11? @glenn-jocher

**Comment by UltralyticsAssistant** [comment]
👋 Hello @JL-52Hertz, thanks for the detailed question and for tagging @glenn-jocher! This is an automated response 🤖—an Ultralytics engineer will review and assist you soon.

For guidance on training workflows and parameters, we recommend starting with the [Docs](https://docs.ultralytics.com/), which include many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) examples covering common setups and options.

Since your request involves specific training setups for COCO and DOTA v1.0 with YOLOv8 and YOLO11, please help us narrow things down so we can share the most relevant info:
-Exact model variants: YOLOv8n/s/m/l/x and YOLO11n/s/m/l/x (and any custom configs)
-Target image size(s), epochs, batch size, and AMP/mixed precision use
-Optimizer/scheduler choices and augmentation settings
-Single vs multi-GPU, DDP setup, and hardware details you plan to use
-Dataset details: versions/splits for COCO and DOTA v1.0, any tiling/rotation or preprocessing
-Any logs or configs you’ve already tried and your target metrics 🎯

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question, please include dataset samples and training logs, and verify you’re following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

You’re welcome to share a minimal command you plan to run, for example:
```bash
# Example CLI templates (customize as needed; see CLI docs)
yolo detect train data=coco.yaml model=yolo11n.pt imgsz=640 epochs=300 batch=64 device=0,1,2,3
yolo detect train data=dota.yaml model=yolov8n.pt imgsz=1024 epochs=300 batch=32 device=0,1 --cache
```
…and we can help align this with the setups used in our benchmarks.

Join the Ultralytics community where it suits you best:
-For real-time chat: [Discord](https://discord.com/invite/ultralytics) 🎧
-For in-depth discussions: [Discourse](https://community.ultralytics.com/)
-For community threads: [Subreddit](https://www.reddit.com/r/Ultralytics/)

## Upgrade

Please upgrade to the latest `ultralytics` (and verify your environment) to ensure parity with current training defaults:
```bash
pip install -U ultralytics
```
Confirm all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [Python>=3.8](https://www.python.org/) environment with [PyTorch>=1.8](https://pytorch.org/get-started/locally/).

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):
-**Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
-**Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
-**Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
-**Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Once you provide the specifics above, we’ll share the closest-matching hardware configuration and training parameter details used for YOLOv8 and YOLO11 on COCO and DOTA v1.0. Thanks! 🙌

**Comment by Y-T-G** [comment]
You can find the training parameters from the model checkpoint:

https://github.com/ultralytics/ultralytics/issues/17048#issuecomment-2424742030
